app/src/test/java/com/codesoom/assignment/controllers/UserControllerApiTest.java

```diff
@@ -0,0 +1,69 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.Utf8MockMvc;
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@Utf8MockMvc
+@DisplayName("User 컨트롤러에서")
+public class UserControllerApiTest {
+    @Autowired MockMvc mockMvc;
+    @Autowired UserRepository userRepository;
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private final String validUserInput = "{\"name\": \"김갑생\", \"email\": \"gabseng@naver.com\", \"password\": \"gabgabhada123\"}";
+
+    @Nested
+    @DisplayName("POST /user 요청은")
```

Comment:
각 단계별 설명을 이어붙여서 하나의 문장이 되는지 검토해 보세요.

```suggestion
    @DisplayName("POST /user 요청을 보낼 때")
```

Reply:
넵!